pages/landing/enterprise-eks/index.html

```diff
@@ -0,0 +1,191 @@
+
+            # Choose from prometheus, datadog, cloudwatch
+            observability: prometheus
+
+            # Configure various common utilities
+            cluster_autoscaler:
+              enabled: true
+
+            sealed_secrets:
+              enabled: true
+
+            aws_load_balancer_controller:
+              enabled: true
+
+            external_dns:
+              enabled: true
+              domain: gruntwork.io
+            </code>
+        </pre>
+      </figure>
+    image_side: right
+    body: |
+      Pick and choose the technologies that you want included in your custom EKS cluster. We
+      support a wide range of popular configurations that include:
+      <ul>
+          <li>Service meshes (Istio, Consul connect, AWS App Mesh)</li>
+          <li>GitOps CI/CD (Flux, Jenkins, Argo)</li>
+          <li>UI dashboards (Porter, Grafana)</li>
+          <li>Observability (Prometheus, Datadog, CloudWatch, ELK)</li>
+          <li>And more (cluster-autoscaler, sealed secrets, AWS Load Balancer Controller, etc)</li>
+      </ul>
+      On top of that, all our enterprise-grade EKS clusters are hardened according to the <a
+      href="https://aws.amazon.com/blogs/containers/introducing-cis-amazon-eks-benchmark/">CIS Amazon EKS
+      Benchmark</a>, regardless of the chosen technologies.
+    logos:
+      - title: CIS
+        src: /assets/img/logos-compliance/cis-benchmark.png
+      - title: EKS
+        src: /assets/img/logos-technologies/logo-eks.png
+
+  - heading: "Step 2: Deploy your EKS cluster"
+    tab_code_preview: site.data.enterprise-landing-page-tabs-code
+    image_side: left
+    body: |
+      Deploy your custom configured EKS cluster. Our friendly user interface can generate code for your existing AWS
+      accounts and VPCs, or you can dive right in to the code. Or, use <a href="https://gruntwork.io/landing-zone-for-aws/">Gruntwork Landing Zone</a> and the
+      <a href="https://gruntwork.io/reference-architecture">Gruntwork Reference Architecture</a> for a complete,
+      end-to-end stack, built with extensibility in mind. No matter what you choose, everything is managed as code with Terraform.
+    logos:
+      - title: Terraform
+        src: /assets/img/landing-heroku-page/terraform-icon.png
+      - title: Docker
+        src: /assets/img/landing-heroku-page/docker-icon.png
+      - title: Packer
+        src: /assets/img/landing-heroku-page/packer-icon.png
+
+  - heading: "Step 3: Set up your app"
+    image: /assets/img/landing-page-enterprise/sample-app-clip.gif
+    image_side: right
+    body: |
+      Use Gruntwork tools to optimize your enterprise apps for EKS. Update your app with cloud-native best practices for
+      packaging, monitoring, logging, service discovery, secrets management, schema migrations, and more. Or customize
+      the optimizations yourself using Gruntwork sample apps as a template. We include sample apps common enterprise languages and
+      frameworks:
```

Comment:
I think a key thing we want to flush out here is that:

1. Your EKS cluster is now deployed and your dev team can now start deploying their apps.
2. The dev team will have a _self-service experience_ for deploying their apps. This experience will include:
    1. A CI / CD pipeline, so deploys happen after commits (GitOps)
    2. A UI that makes it easy for devs to understand what's happening in the cluster
    3. Reference applications that show cloud-native best practices
    4. Perhaps battle-tested reference Helm charts? Core idea is devs can deploy a new app without having to ask Ops team, but still have all the right security & reliability settings built-in.
    5. Detailed guides that walk the devs through each step of packaging, configuring, and deploying their app.

I'm not sure how to capture this in these steps, but the self-service dev experience is a [key differentiator](https://www.notion.so/gruntwork/Enterprise-Enterprise-EKS-344f903153b448d2a93fd9edc82adb42#677be35f65f44e5c86ca1214ed252248), so we want to make sure it's visible.

Reply:
I reorganized this with this in mind. I think the biggest challenge here is how to be concise while highlighting these points.